src/app/(main)/wines/page.tsx

```
@@ -91,6 +92,15 @@ const WinesPage = () => {
           <WinePriceSlider />
           <Button type='submit'>Submit</Button>
         </form>
+        <button onClick={() => open('wine-register')}>와인 등록</button>
+        <Modal
+          modalId='wine-register'
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)